src/main/export.ts

```diff
@@ -12,22 +12,27 @@ const secondToTimestamp: (num: number) => string = (num) => {
     .join(':');
 };
 
-const constructEDL: (title: string, transcription: Transcription) => string = (
-  title,
-  transcription
-) => {
+const constructEDL: (
+  title: string,
```

Comment:
Instead of passing three distinct parts of a Project type, maybe type this instead as `Partial<Project, 'name' | 'transcription' | 'mediaFilePath'> and then just pass the whole project as an argument (typescript will block you from accessing other fields you didn't declare)